changelog/updates/2022-09-14-libxml2.md

```diff
@@ -0,0 +1,2 @@
+- libxml2 ([2.10.2](https://gitlab.gnome.org/GNOME/libxml2/-/tags/v2.10.2))
+- libxslt ([1.1.35](https://gitlab.gnome.org/GNOME/libxslt/-/tags/v1.1.35))
```

Comment:
How about `- SDK: libxslt ...`, as libxslt is included only in the SDK etc.?

Reply:
Sure.